test-resources/bin/util

```diff
@@ -447,51 +447,15 @@ run_helm() {
   helm_install_command+=" ${OPERATOR_HELM_CHART:-helm-chart/dash0-operator}"
   echo Helm install command:
   echo "$helm_install_command"
-  $helm_install_command
-
-  wait_for_operator_manager_and_webhook
```

Comment:
All of the logic in wait_for_operator_manager_and_webhook is no longer necessary, because that is now handled by the extended readiness check and the Helm post-install hook. (Because we are now also passing `--wait` to `helm install`.)